src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -237,6 +237,8 @@ tests:
                 value: 0x0001
               - name: "SceneID"
                 value: 0x01
+      response:
+          error: ''
```

Comment:
This should list the error expected based on the spec, if this is supposed to error, no?

That said, per test plan this is not supposed to error, so why is this being added?

Reply:
Empty error is the equivalent of a check for success, if any error show up, the test will fail because this expects none.

Reply:
The test will fail on error if there is no `response:` section, no?

Reply:
Indeed

Reply:
Removed it.